### PR TITLE
Agency Provider Structure Tweaks

### LINKED ISSUE #828: Final Agency/Provider Questions
### Is your feature request related to a problem? Please describe.

Some minor open questions for discussions around the agency and provider unification work in #796 and #826.

### Describe the solution you'd like

To be discussed here and resolved in a future PR if needed:

1. Can we rename the Agency vehicle events endpoint from /vehicles/events to just /events, and /vehicles/telemetry to just /telemetry? [See comment](https://github.com/openmobilityfoundation/mobility-data-specification/pull/826#discussion_r1088990579)
2. Where should the Vehicle Status object be linked to? [See comment](https://github.com/openmobilityfoundation/mobility-data-specification/pull/826#issuecomment-1402669982)
3. Remove routes from Provider and ensure it's clear in telemetry

### Is this a breaking change

 * Yes, breaking

### Impacted Spec

For which spec is this feature being requested?

 * `agency`
 * `provider`

### Describe alternatives you've considered

Leave as is.

### Additional context

N/A

### PR DESCRIPTION
**Pull request by schnuerle**
## Explain pull request

Final work for unification of Agency and Provider. 

## Is this a breaking change

* Yes, breaking

## Impacted Spec

Which spec(s) will this pull request impact?

* `agency`
* `provider`

## Additional context

See details at #828

### CODE REVIEW

**Comment by schnuerle** [comment]
Where has [stop based geo data](https://github.com/openmobilityfoundation/mobility-data-specification/blob/main/general-information.md#stop-based-geographic-data) gone to? It's [referenced](https://github.com/openmobilityfoundation/mobility-data-specification/tree/feature-agency-provider-tweaks/provider#telemetry) in Provider in telemetry and events. Instead does GPS in data types just get an optional stop_id?

File: agency/README.md [hunk 1]
```diff
@@ -100,17 +101,52 @@ _No content returned on vehicle not found._
...
+
+Path Params:
+
+| Param        | Type | Required/Optional | Description                                 |
+| ------------ | ---- | ----------------- | ------------------------------------------- |
+| `device_id`  | UUID | Optional          | If provided, retrieve the specified vehicle |
+
+200 Success Response:
+
+If `device_id` is specified, `GET` will return an array with a single vehicle record, otherwise it will be a list of vehicle records with pagination details per the [JSON API](https://jsonapi.org/format/#fetching-pagination) spec:
```
Comment (marie-x) [inline agency/README.md:119]: vehicle status record

File: agency/README.md [hunk 2]
```diff
@@ -100,17 +101,52 @@ _No content returned on vehicle not found._
...
+
+```json
+{
+    "vehicles": [ ... ]
```
Comment (marie-x) [inline agency/README.md:123]: "vehicles_status":

File: provider/README.md [hunk 3]
```diff
@@ -214,10 +214,47 @@ _No content returned on vehicle not found._
...
 
+### Vehicle Status
+
+The `/vehicles/status` endpoint returns the specified vehicle (if a device_id is provided) or a list of known vehicles. Contains specific vehicle properties that are updated frequently.
+
+**Endpoint:** `/vehicles/status/{device_id}`  
+**Method:** `GET`  
+**[Beta feature][beta]:** No (as of 1.2.0)  
+**Schema:** [`vehicles-status` schema][vehicles-status-schema]  
+**`data` Payload:** `{ "vehicles": [] }`, an array of [Vehicle Status](vehicle-status) objects
```
Comment (marie-x) [inline provider/README.md:225]: "vehicles_status":

File: provider/README.md [hunk 4]
```diff
@@ -214,10 +214,47 @@ _No content returned on vehicle not found._
 
 [Top][toc]
 
+### Vehicle Status
+
+The `/vehicles/status` endpoint returns the specified vehicle (if a device_id is provided) or a list of known vehicles. Contains specific vehicle properties that are updated frequently.
```
Comment (marie-x) [inline provider/README.md:219]: vehicle status records

File: provider/README.md [hunk 5]
```diff
@@ -214,10 +214,47 @@ _No content returned on vehicle not found._
...
+
+Path Params:
+
+| Param        | Type | Required/Optional | Description                                 |
+| ------------ | ---- | ----------------- | ------------------------------------------- |
+| `device_id`  | UUID | Optional          | If provided, retrieve the specified vehicle |
+
+200 Success Response:
+
+If `device_id` is specified, `GET` will return an array with a single vehicle record, otherwise it will be a list of vehicle records with pagination details per the [JSON API](https://jsonapi.org/format/#fetching-pagination) spec:
```
Comment (marie-x) [inline provider/README.md:235]: vehicle status record

**Review by marie-x — DISMISSED** [review]
minor clarifications but LGTM

**Pull request merged**